server/pulp/server/webservices/views/consumer_groups.py

```diff
@@ -10,7 +10,9 @@
 from pulp.server.managers.consumer.group.cud import bind, unbind
 from pulp.server.webservices.controllers.decorators import auth_required
 from pulp.server.webservices.views.util import (generate_json_response,
-                                                generate_json_response_with_pulp_encoder)
+                                                generate_json_response_with_pulp_encoder,
+                                                json_body_allow_empty,
```

Comment:
Maybe s/json_body_allow_empty/json_body_required_allow_empty/   At face value it seems that it just allows empty with no requirement enforced.